arm/CHANGELOG.md

```diff
@@ -21,9 +21,9 @@ updated in this release):
 - *arm/notificationhubs 2014-09-01*
 - arm/redis 2015-08-01
 - *arm/resources/authorization 2015-01-01*
-- *arm/resources/features 2014-08-01-preview*
```

Comment:
I think I need to update this (v0.5.0) in version template and regenerate the SDK. Right?

const ( 
26 	major = "0" 
27 	minor = "4" 
28 	patch = "0" 
29 	// Always begin a "tag" with a dash (as per http://semver.org) 
30 	tag             = "-beta" 
31 	semVerFormat    = "%s.%s.%s%s" 
32 	userAgentFormat = "Azure-SDK-for-Go/%s;Package arm/%s;API %s" 
33 )